internal/client/cloudfoundry/client.go

```diff
@@ -517,41 +517,25 @@ func (cfc *CFClient) getV2ApplicationsByPage(page int) ([]CFApplication, int, er
 
 func (cfc *CFClient) GetV2Orgs() ([]cfclient.Org, error) {
 	// NOTE: this is used by the OrgCollector
-	var allOrgs []cfclient.Org
 	query := url.Values{}
-	query.Set("results-per-page", "100")
+	query.Set("results-per-page", "1")
```

Comment:
Is this on purpose ?

Reply:
Whoops, good catch :) I did some testing here... sorry, will fix :)

Reply:
Fixed.